api/test/h3-data/mocks/create-fake-h3-data.ts

```diff
@@ -41,8 +45,8 @@ export const createFakeH3Data = async (
 };
 
 export const dropFakeH3Data = async (h3TableNames: string[]): Promise<void> => {
-  for (const h3tableName of h3TableNames) {
-    await getManager().query(`DROP TABLE IF EXISTS ${h3tableName};`);
+  for (const h3TableName of h3TableNames) {
+    await getManager().query(`DROP TABLE IF EXISTS ${snakeCase(h3TableName)};`);
```

Comment:
Shouldn't this be:

```suggestion
    await getManager().query(`DROP TABLE IF EXISTS ${formattedTableName)};`);
```

Reply:
formattedTableName does not exist in the scope of this function